hubmap/frontend/templates/listall.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
```

Comment:
This seems to be doing the same thing as entities.html: Do we need both?
Naming: again, please do not run words together in names. Use a dash, underscore, or capital letter, depending on context. This has come up multiple times.

Reply:
actually this is fixed in another commit as per Matt's RC (**R**eview **C**omments).  marking this as resolved.